setup.cfg

```diff
@@ -65,19 +69,41 @@ all =
 
 
 [options.package_data]
-* = 
+* =
     peakdet/tests/data/*
 
 [flake8]
 exclude=
     *build/
-    heuristics
     tests
+    versioneer.py
 ignore = E126, E203, E402, W503
 max-line-length = 99
```

Comment:
Sorry, you are right about black in setup.cfg.

I got confused with extended Flake8 settings to match Black:

ignore = E126, E402, W503, F401, F811
max-line-length = 88
extend-ignore = E203, E501
extend-select = B950

Reply:
Done